### PR TITLE
feat(testing) Testing framework chapter 1, asserts and comments

### PR DESCRIPTION
**Pull request by monoid**
1. Rename `seq_result` -> `seq_ok`, and add `seq_err`.
2. Accept `;;` and longer comments in the sexp parser.

### CODE REVIEW

**Review by mikevoronov — APPROVED** [review]
utACK

File: crates/testing-framework/src/execution/mod.rs [hunk 1]
```diff
@@ -322,6 +325,137 @@ mod tests {
...
+                    executed_state::scalar(json!(["peer2", "peer3"])),
+                    executed_state::scalar(json!(42)),
+                    executed_state::scalar(json!(43)),
+                ]
+            );
+        }
+    }
+
+    #[test]
+    fn test_seq_error() {
```
Comment (ValeryAntopol) [inline crates/testing-framework/src/execution/mod.rs:384]: This test looks too large. Maybe split is in several ones?

File: crates/testing-framework/src/services/results.rs [hunk 2]
```diff
@@ -74,6 +77,37 @@ fn seq_result_closure(call_map: HashMap<String, serde_json::Value>) -> CallServi
...
+                    call_num_str, call_map
+                )
+            })
+            .clone()
+    })
+}
+
+fn map_service_closure(map: HashMap<String, serde_json::Value>) -> CallServiceClosure {
+    Box::new(move |args| {
+        let repr = match &args.arguments[0] {
```
Comment (ValeryAntopol) [inline crates/testing-framework/src/services/results.rs:103]: What if args are empty? I suggest panic with a good error message.

**Review by ValeryAntopol — APPROVED** [review]
utACK

I think comments fixes do not require additional review

File: crates/testing-framework/src/execution/mod.rs (hunk 1 above)
Comment (monoid) [reply crates/testing-framework/src/execution/mod.rs:399]: It going to be several long tests.

**Pull request merged**